04/Task04.js

```diff
@@ -19,7 +20,17 @@ const Task04 = () => {
                 </RBTabs>
             </Col>
             <Col>
-                Tabs!
+                <Tabs defaultActiveKey="profile" id="uncontrolled-tab-example">
+                    <Tab eventKey="home" title="Home">
```

Comment:
Przy przełączaniu okno niepotrzebnie przeskakuje i nie widać "zaznaczonej" zakładki (brakuje obramowania) - warto by było to poprawić ;)